nchelpers/__init__.py

```diff
@@ -500,6 +537,19 @@ def is_hydromodel_iobs_output(self):
         return self.is_hydromodel_output and self.forcing_type == 'gridded observations'
 
     @property
+    def model_type(self):
+        """String indicating what type of model the file contains.
+        Supports modelmeta/mm_cataloguer/index_netcdf.py.
+        Really rudimentary decision making about model type.
```

Comment:
This is how the script was written originally, so it's fine for this PR. But we should probably log an issue against the function to improve it (i.e. cover all of the use cases that we'll be running it against).